data/campaigns/The_Rise_Of_Wesnoth/units/Wesfolk_Outcast.cfg

```diff
@@ -18,7 +18,7 @@
     advances_to=Wesfolk Lady
     cost=18
     usage=mixed fighter
-    description= _ "Born in the Wesfolk aristocracy, she and her people were excluded from society by their lords, who betrayed their loyalty when a war against Haldric’s people was being lost. This outcast still conserves her nobleness in her veins and on the battlefield she has earned valuable experience, which she can lend to her people for turning a fight in their favor."
+    description= _ "Born in the Wesfolk aristocracy, she and her people were excluded from society by their lords, who betrayed their loyalty when a war against Haldric’s people was being lost. This outcast still conserves her nobleness in her veins, and on the battlefield she has earned valuable experience, which she can lend to her people for turning a fight in their favor."
```

Comment:
Was the single comma that necessary?

Reply:
![image](https://user-images.githubusercontent.com/41433185/135712743-c62c9906-42a1-4098-bc01-a66715046fb5.png)
independent clause.

Reply:
I agree with this comma. Without it, it lends to parsing it like this, which then leads you into a grammatical dead end (indicated by the asterisk):

> This outcast still conserves her nobleness [in her veins] and [on the battlefield] *she has earned valuable experience, which she can lend to her people for turning a fight in their favor.

There's one other way I can see to solve this:

> This outcast still conserves her nobleness in her veins and on the battlefield; she has earned valuable experience, which she can lend to her people for turning a fight in their favor.

However, I think the meaning implied by the added comma is probably closer to the intention than the different meaning conveyed by adding that semicolon.